docker/haveno/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM openjdk:11
+
+RUN set -ex && \
+    apt update && \
+    apt --no-install-recommends --yes install \
+        make \
+        git
+
+RUN set -ex && adduser --system --group --disabled-password haveno && \
+    mkdir -p /home/haveno && \
+    chown -R haveno:haveno /home/haveno
+
+USER haveno
+
+RUN set -ex && git clone -b docker https://github.com/napoly/haveno.git /home/haveno/haveno
```

Comment:
don't forget to remove branch from here before merging and merging docker on the haveno side